components/list/list-item-mixin.js

```diff
@@ -67,6 +67,11 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			 * @type {array}
 			 */
 			breakpoints: { type: Array },
+			/**
+			 * A color indicator to appear at the beginning of a list item. Expected value is a valid color hex code (e.g., '#006fbf').
```

Comment:
Hex values could also include the alpha channel. ex. `#RRGGBBAA` for semi-transparent colors. I'm not sure if they will try to use semi-transparent colors, or whether we want to actively stop them from doing so. That's probably a follow-up for the designers, and depending on what they say we might want to update this doc comment.

Related: Although it is more work for the list/list-items to do, I think we need the list components to automatically adjust spacing based on whether other items in the same list have color indicators rendered. Allowing consumers to have any influence over this multiplies the nuttiness of all the spacing permutations that Jeff is trying to sort out. So, assuming that, the consumer shouldn't ever have a need for a fully transparent color (`transparent`, `rgba(R,G,B,0)`, `#RRGGBB00`).